Bug: http://bugzilla.gnome.org/show_bug.cgi?id=126051

This will be obsolete if/when upstream has PolicyKit integration
Index: gnome-system-log-3.3.1/data/gnome-system-log.desktop.in.in
===================================================================
--- gnome-system-log-3.3.1.orig/data/gnome-system-log.desktop.in.in	2011-10-04 17:13:20.000000000 +0200
+++ gnome-system-log-3.3.1/data/gnome-system-log.desktop.in.in	2012-03-13 00:40:30.111656072 +0100
@@ -1,11 +1,11 @@
 [Desktop Entry]
 _Name=Log File Viewer
 _Comment=View or monitor system log files
-Exec=gnome-system-log
+Exec=gnome-system-log-pkexec
 Icon=logview
 Terminal=false
 Type=Application
-StartupNotify=true
+StartupNotify=false
 Categories=GTK;GNOME;System;Monitor;
 NotShowIn=KDE;
 X-GNOME-DocPath=gnome-system-log/gnome-system-log.xml
